openslides_backend/action/actions/meeting/export_helper.py

```diff
@@ -62,30 +66,47 @@ def export_meeting(datastore: DatastoreService, meeting_id: int) -> Dict[str, An
         else:
             export[str(collection)] = {}
 
+    user_ids = set()
```

Comment:
user_ids are initialized in line 44 (in a better way)

Reply:
Remove this surplus user_ids initialize lines. If the tests still work, I will merge it.